crates/codegen/parser/runtime/src/templates/kinds.rs.jinja2

```diff
@@ -33,6 +33,34 @@ impl RuleKind {
     }
 }
 
+#[derive(
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    serde::Serialize,
+    strum_macros::AsRefStr,
+    strum_macros::Display,
+    strum_macros::EnumString,
+)]
+#[strum(serialize_all = "snake_case")]
+#[cfg_attr(feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "kinds"))]
+#[cfg_attr(not(feature = "slang_napi_interfaces"), derive(Clone, Copy))]
+pub enum FieldName {
```

Comment:
Done in https://github.com/NomicFoundation/slang/pull/713/commits/8373fd8f33f29183cd36b4cf723f8bda0679d834; I couldn't use `FieldName` directly due to build issues, so I've defined a subset and added a marker, so that it doesn't go out of sync.